webapp/components/post_view/post_focus_view_controller.jsx

```diff
@@ -103,7 +107,15 @@ export default class PostFocusView extends React.Component {
     }
 
     onPreferenceChange() {
+        const focusedPostId = PostStore.getFocusedPostId();
+        if (focusedPostId == null) {
+            return;
```

Comment:
Just checking but this view will always have a focused post? If not, then it looks like it would be possible to miss changes in the flagged posts with this return